apps/site/components/Common/Partners/PartnerButton/index.tsx

```diff
@@ -0,0 +1,26 @@
+import Button from '#site/components/Common/Button';
+
+import type { AnchorHTMLAttributes, FC } from 'react';
+
+import style from './index.module.css';
+
+type PartnerProps = AnchorHTMLAttributes<HTMLAnchorElement> & {
+  size?: 'big' | 'small';
```

Comment:
```suggestion
  size?: 'small' | 'large';
```

I think small and large would be better naming choices, since we use similar naming conventions in our other components as well